src/status_im2/common/home/view.cljs

```diff
@@ -30,84 +29,65 @@
      :override-background-color (when (and dark? default?)
                                   colors/neutral-90)}))
 
-(defn- base-button
-  [icon on-press accessibility-label button-common-props]
-  [quo/button
-   (merge
-    {:on-press            on-press
-     :accessibility-label accessibility-label}
-    button-common-props)
-   icon])
+(defn- unread-indicator
+  []
+  (let [unread-count (rf/sub [:activity-center/unread-count])
+        indicator    (rf/sub [:activity-center/unread-indicator])
+        unread-type  (case indicator
+                       :unread-indicator/seen :grey
+                       :unread-indicator/new  :default
+                       nil)]
+    (when (pos? unread-count)
+      [quo/counter
+       {:type  unread-type
+        :style (style/unread-indicator unread-count)}
+       unread-count])))
+
+(defn- left-section
+  [{:keys [avatar]}]
+  [rn/touchable-without-feedback {:on-press #(rf/dispatch [:navigate-to :my-profile])}
+   [rn/view
+    {:accessibility-label :open-profile
+     :style               style/left-section}
+    [quo/user-avatar
+     (merge {:status-indicator? true
+             :size              :small}
+            avatar)]]])
+
+(defn- right-section
+  [{:keys [button-type hide-search]}]
+  (let [button-common-props (get-button-common-props button-type)]
+    [rn/view {:style style/right-section}
+     (when-not hide-search
+       [quo/button
+        (merge {:accessibility-label :open-search-button}
```

Comment:
What do you think about using 
```clojure
(assoc map :accessibility-label :a-key)
```

instead of
```clojure
(merge {:accessibility-label :a-key} map)
```
?

I've noted that the map returned by `get-buttom-common-props` doesn't have an `:accessibility-label` key, so I'd say we can `assoc` the `:accessibility-label` as I suggested without any problems.

Reply:
I prefer your suggestion, it's less expensive too.